nucypher/config/constants.py

```diff
@@ -14,20 +14,24 @@
 You should have received a copy of the GNU General Public License
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
+
+
 from collections import namedtuple
 from os.path import abspath, dirname
 
 from appdirs import AppDirs
 
 import nucypher
 
-PACKAGE_NAME = 'nucypher'
 
 # Base Filepaths
 BASE_DIR = abspath(dirname(dirname(nucypher.__file__)))
 PROJECT_ROOT = abspath(dirname(nucypher.__file__))
-APP_DIR = AppDirs(PACKAGE_NAME, "NuCypher")
+
+# User Application Filepaths
+APP_DIR = AppDirs(nucypher.__title__, nucypher.__author__)
 DEFAULT_CONFIG_ROOT = APP_DIR.user_data_dir
+USER_LOG_DIR = APP_DIR.user_log_dir
```

Comment:
How can we inform the user of this location? I had to run this code to find out where OS X puts this.

Reply:
Perhaps it can be included in `nucypher ursula view` or `nucypher configure view`?